SYCL/ESIMD/api/functional/value.hpp

```diff
@@ -59,6 +59,26 @@ sycl::half half_from_bytes(unsigned char hi, unsigned char lo) {
   return esimd_test::bit_cast<sycl::half>(out);
 }
 
+// Constructs std::vector with from provided vectors.
```

Comment:
```suggestion
// Constructs std::vector from provided vectors.
```
But the description does not reflect the (very complex) function logic - please fix.

Reply:
I've applied this comment in this commit: 00693ecea35d0a2935c72bb247a7a9835f965a42